packages/expo-font/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### 🐛 Bug fixes
 
+- Fixed timeout on IE [#8652](https://github.com/expo/expo/pull/8652)
```

Comment:
Does it fix fonts not being loaded or fix timeout when loading fonts (they still don't load even so)?
```suggestion
- Fixed fonts not being loaded in Internet Explorer ([#8652](https://github.com/expo/expo/pull/8652) by [@d4rky-pl](https://github.com/d4rky-pl))
```

Reply:
Fixes both loading and the timeout I guess (they load and don't timeout). I based the message on the same one but for Firefox (which uses exactly the same fix)

Reply:
Yeah, that change log has been introduced after the actual fix as a history log, the common format we use is something more like the text proposed, eg.

https://github.com/expo/expo/blob/52217f1a59f2d02ebbb4eeeea3e24064cd544980/packages/expo-notifications/CHANGELOG.md#L39